source/clear-linux/guides/tooling/mixer.rst

```diff
@@ -378,21 +394,28 @@ mix version 10 (from which the image was built), to mix version 20.
       swupd bundle-list
       swupd bundle-list -a
 
-   .. note::
+#. Now we will add the `editors` bundle that we modified.
+
+   .. code-block:: bash
+
+      swupd bundle add editors
+
+#. Try to start `joe` editor.  It should not appear because we removed it
+   from the original `editors` bundle.
 
-      You cannot see the curl bundle that you added in Example 2 because
-      your mix is still on version 10.
+#. Next we will update from version 10 to 20 to capture the newly
+   available bundles.
 
-   Check for updates. You should see that version 20 is available. Use
-   :command:`swupd` to update your mix:
+#. Check to see that version 20 is available. Use :command:`swupd`
```

Comment:
I think line 406 and 409 are saying the same thing.  I think you should remove 409 and just put the code-block under under line 406.

Reply:
Done